src/qibocal/protocols/rabi/amplitude_frequency.py

```diff
@@ -234,10 +235,16 @@ def _plot(
             row=1,
             col=1,
         )
+
+        if data.rx90:
+            pulse_name = "Pi-half pulse"
+        else:
+            pulse_name = "Pi pulse"
```

Comment:
In these cases, it is especially convenient to use the ternary `if` operator (exact same of the `... ? ... : ...` construct in C)
```suggestion
        pulse_name = "Pi-half pulse" if data.rx90 else "Pi pulse"
```